training-maquetado/application.css

```diff
@@ -0,0 +1,115 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto");
+* {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
+  padding: 0; }
+
+h1,
```

Comment:
this application.css should be ignored using .gitignore, only the scss folder should be available